traefik-values-01.yaml

```diff
@@ -0,0 +1,16 @@
+ports:
+  web:
+    nodePort: 30000
+  websecure:
+    nodePort: 30001
+logs:
+  access:
+    enabled: true
+ingressRoute:
+  dashboard:
+    enabled: true
+    # Custom match rule with host domain
+    matchRule: Host(`dashboard.docker.localhost`)
```

Comment:
The `nsswitch.conf` needs to be configured with  [myhostname](https://www.freedesktop.org/software/systemd/man/nss-myhostname.html) module enabled for `host`.

Reply:
I wasn't aware of that. Thanks. I updated accordingly.